apps/codeforafrica/content/projects/sensors-africa.md

```diff
@@ -0,0 +1,29 @@
+---
+id: 96d3e8eb-a760-44d5-8128-4651ffe3aea1
+name: sensors.AFRICA
+tagLine: We are here to give you actionable information about the quality of
+  your city's air, water, sound and radiation levels.
```

Comment:
This needs to be very short... We should also may be clip it in code around 25 characters?
```suggestion
tagLine: Your city's air quality.
```